core/templates/dev/head/galleries/Gallery.js

```diff
@@ -218,7 +218,7 @@ oppia.controller('Gallery', [
     });
 
     // Below is the width of each tile.
-    var TILE_DISPLAY_WIDTH = 211.953;
+    var TILE_DISPLAY_WIDTH = 208;
```

Comment:
(In fact, if possible, you may want to do a dual reference, such that each points to the other -- that way, when one is updated, the other will be.)